frontend/src/interfaces/pagedItems.tsx

```diff
@@ -5,5 +5,5 @@ export interface IPagedItems {
   page: number;
   quantity: number;
   total: number;
-  items: [];
+  items: any[];
```

Comment:
Make the `IPageItems` generic so that items is strongly typed.

Reply:
Good idea